cmd/zpool/zpool_main.c

```diff
@@ -337,8 +339,8 @@ get_usage(zpool_help_t idx)
 	case HELP_SCRUB:
 		return (gettext("\tscrub [-s] <pool> ...\n"));
 	case HELP_STATUS:
-		return (gettext("\tstatus [-c CMD] [-gLPvxD] [-T d|u] [pool]"
-		    " ... [interval [count]]\n"));
+		return (gettext("\tstatus [-c [script1,script2,...]] [-gLPvxD]"
```

Comment:
should this be something like `return (gettext("\tstatus [-c script1[,script2,...]] [-gLPvxD]"`? You need to specify at least one script to run? The list of multiple scripts to run is optional part.